set.mm

```diff
@@ -12253,13 +12253,15 @@ This definition (in the form of ~ dfifp2 ) appears in Section II.24 of
     QUITUHUCTUIABIJUFUAUGUBAKBKLMNTUBUAOR $.
 
   $( Introduce a right anti-conjunct to both sides of a logical equivalence.
```

Comment:
I checked out the master of 01-01-2018 and looked into AH's mathbox.  It seems no shortening took place, but a renaming.  Here is what I found:
```
$( set.mm - Version of 2-Jan-2018 $)
```
...
```
$(
#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#
                   Mathbox for Anthony Hart
#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#
$)

$(
=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
        Propositional Calculus
=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
$)
```
...
```
  $( Constructor theorem for ` -/\ ` .  (Contributed by Anthony Hart,
     1-Sep-2011.) $)
  nabi1 $p |- ( ( ph <-> ps ) -> ( ( ph -/\ ch ) <-> ( ps -/\ ch ) ) ) $=
    ( wb wa wn wnan anbi1 notbid df-nan 3bitr4g ) ABDZACEZFBCEZFACGBCGLMNABCHIA
    CJBCJK $.

  $( Constructor theorem for ` -/\ ` .  (Contributed by Anthony Hart,
     1-Sep-2011.) $)
  nabi2 $p |- ( ( ph <-> ps ) -> ( ( ch -/\ ph ) <-> ( ch -/\ ps ) ) ) $=
    ( wb wa wn wnan anbi2 notbid df-nan 3bitr4g ) ABDZCAEZFCBEZFCAGCBGLMNABCHIC
    AJCBJK $.
```

Reply:
Thanks for the enquiry @wlammen. What do you recommend ? Remove these two "proof shortened" tags ?